styles/_theme.scss

```diff
@@ -104,13 +104,20 @@ input[type=range][class~="clrS"] {
     border-color: $border;
   }
 
+  .trumbowyg-box,
```

Comment:
Removing the top border is problematic for the focus state:

before ==>

![image](https://cloud.githubusercontent.com/assets/794517/21702511/a7a4bf64-d360-11e6-8470-03f139389bab.png)

after ==>

![image](https://cloud.githubusercontent.com/assets/794517/21702522/b3f7f6a0-d360-11e6-8de7-02ac5ff7f9ec.png)

Is there a way you could remove it without it messing up the focus state? Maybe on focus the top border can come from the bottom border of the nav bar?